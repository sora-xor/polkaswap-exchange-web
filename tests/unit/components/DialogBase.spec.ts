// import { shallowMount } from '@vue/test-utils'

// import DialogBase from '@/components/DialogBase.vue'

// describe('DialogBase.vue', () => {
//   it('should renders correctly', () => {
//     const wrapper = shallowMount(DialogBase, {
//       propsData: {
//         visible: true,
//         customClass: 'custom-dialog-class',
//         title: 'Dialog title'
//       }
//     })
//     expect(wrapper.element).toMatchSnapshot()
//   })
// })

describe('DialogBase test', () => {
  test('Temporary test', () => {
    expect(true).toEqual(true);
  });
});
