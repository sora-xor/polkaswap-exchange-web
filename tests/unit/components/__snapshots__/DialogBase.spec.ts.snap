// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogBase.vue should renders correctly 1`] = `
<s-dialog
  border-radius="medium"
  class="dialog-wrapper"
  custom-class="custom-dialog-class"
  title="Dialog title"
  top="80px"
  width="496px"
>
   
  <template />
</s-dialog>
`;
